src/content/docs/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,188 @@
+
+
+## Upgrade to the latest version
+
+Update your project's version of Astro and all official integrations to the latest versions with one command using your package manager:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # Upgrade Astro and official integrations together
+  npx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Upgrade Astro and official integrations together
+  pnpm dlx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Upgrade Astro and official integrations together
+  yarn dlx @astrojs/upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+### Manual Upgrading
+
+To update Astro and integrations to their current versions manually, use the appropriate command for your package manager.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # Example: upgrade Astro with React and Tailwind integrations
+  npm install astro@latest @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Example: upgrade Astro with React and Tailwind integrations
+  pnpm add astro@latest @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Example: upgrade Astro with React and Tailwind integrations
+  yarn add astro@latest @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Install a specific version number
+
+To install a specific [version of Astro](https://www.npmjs.com/package/astro?activeTab=versions) or integrations, use the appropriate command for your package manager.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install astro@4.5.3 @astrojs/react@3.0.10
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add astro@4.5.3 @astrojs/react@3.0.10
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add astro@4.5.3 @astrojs/react@3.0.10
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+## Documentation updates
+
+This documentation is updated for each [minor release](#minor-changes) and [major version release](#major-changes). When new features are added, or existing usage changes, the docs will update to reflect the **current behavior of Astro**. If your project is not updated, then you may notice some behaviors do not match the up-to-date documentation. 
+
+New features are added to docs with the specific version number in which they were added. This means that if you have not updated to the latest release of Astro, some documented features may be unavailable. Always check the `Added in:` version number and make sure your project is updated before attempting to use new features!
```

Comment:
Possible improvement to the wording of this sentence:
```suggestion
New features are added to docs with the specific version number in which they were added. This means that if you have not updated to the latest release of Astro, some documented features may be unavailable. Always check the `Added in:` version number to ensure your project version is compatible with the features you want to use.
```

Reply:
I think both versions are pretty comparable! In this specific context, on a page about upgrading/updating and how we document *new* features, I *do* slightly prefer the extra nudge to update their project to use new features, but I don't have a *strong* preference. Yours uses *slightly* move elevated language, which can be *slightly* more work to translate properly, or a bit of friction for people reading in their second language, but it's certainly a fine edit!

Is there a problem you're attempting to fix here, so I know the motivation and can better decide between the two wordings?

Reply:
My main goal with this change was to be a bit more clear about that new features meant. Someone could be a version behind from where a feature was introduced but that feature doesn't necessarily have to be in the latest version of Astro. Rewording it to `compatible with...` makes it a bit more clear in my mind.

This is a very minor change and as you mentioned, the two versions are very comparable. I think we could go either way.

Reply:
s nAgreed, they are comparable and my point here was to keep with the theme of upgrading/new features. So, there's nothing wrong with your suggestion, and on a more generic page I probably would have used it! That's why I asked exactly what problem you were trying to solve, to see what I might be missing.

I think I'll keep this nuance as is for now, but I do appreciate these checks for clarity! :purple_heart: